### PR TITLE
DES references

### PR DESCRIPTION
**Pull request by mpaillassa**
I am starting this PR to update the DES survey even though the HSC PR #32 is not yet merged as we are waiting for the last HSC obscuration details.

Note that the CTIO website URL has recently changed from `ctio.noao.edu` to `ctio.noirlab.edu` so that the main DES survey parameter reference links in the corresponding issues are dead.

For the filter parameters, I have chosen to use the [DR1 paper](https://arxiv.org/pdf/1801.03181.pdf) as it contains the sky_brightness, exp_time and psf_fwhm in the same place (we use speclite for zeropoints). We may use the [DR3](https://arxiv.org/pdf/2011.03407.pdf) values but the sky brightness is given in e-/pixel so it would need to be converted to mag/arcsec^2.
Also, in practice, exposure times are 90s (for DR1) and those exposures are coadded by 10 to make 900s coadds so I am not sure if we should set 90s or 900s as exposure time. As previous values were about 800-1000s, I assumed we should use 900s.

### CODE REVIEW

**Comment by ismael-mendoza** [comment]
Hi @aboucaud  and @mpaillassa - do you also want to include the `central_wavelength` in every filter of each survey? Currently only "Rubin" has this specified.

**Comment by mpaillassa** [comment]
Hi @ismael-mendoza. Yes, I remember that we use the same Rubin/LSST central wavelength for all surveys in BTK but it would be nice to have a central wavelength for each specific survey filter. I guess this is to be managed with #36.

For DES we can find information [here](https://speclite.readthedocs.io/en/latest/filters.html#decam-2014-filters). Especially, I guess we can compute the central wavelength thanks to the Excel spreadsheet but what is the exact definition of the central wavelength ? Is it the filter response weighted sum of wavelengths through the bandpass ?

**Comment by ismael-mendoza** [comment]
Thanks @mpaillassa for the info, I'll talk to Javi about how they computed it in WeakLensingDeblending and let you know. The relevant line is [here](https://github.com/LSSTDESC/WeakLensingDeblending/blob/228c6655d63de9edd9bf2c8530f99199ee47fc5e/descwl/survey.py#L170) but the link mentioned is broken.

Feel free to proceed with this PR if possible without the central_wavelength information and we will manage it in a different PR along #36

**Review by aboucaud — APPROVED** [review]
Looking good !

I find it very strange that the PSF FWHM values were that much off.. 
Thanks for seeing this through.

**Comment by aboucaud** [comment]
@mpaillassa are you ok with resolving the existing conflicts or do you want me to do it ?

**Comment by mpaillassa** [comment]
@aboucaud It's ok I just did it.

File: galcheat/data/DES.yaml [hunk 1]
```diff
@@ -1,34 +1,59 @@
...
+    psf_fwhm: 0.88
   z:
     name: "z"
-    sky_brightness: 18.7
-    exp_time: 800
+    sky_brightness: 18.72
+    exp_time: 900
     zeropoint: 26.58
-    psf_fwhm: 1.12
+    psf_fwhm: 0.84
```
Comment (aboucaud) [inline galcheat/data/DES.yaml:59]:
```suggestion
    psf_fwhm: 0.84

```

**Comment by aboucaud** [comment]
Thanks a lot @mpaillassa !

**Pull request merged**